masterlist.yaml

```
@@ -4750,9 +4750,17 @@ plugins:
         util: '[TES4Edit v4.0.3g](https://www.nexusmods.com/oblivion/mods/11536)'
         itm: 16
 
+  - name: 'Oblivion Content Restoration Project( -- Knights)?\.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/45909/'
+        name: 'Oblivion Content Restoration Project'
   - name: 'Oblivion Content Restoration Project.esp'
-    url: [ 'https://www.nexusmods.com/oblivion/mods/45909' ]
     req: [ *OBSE ]
+    inc: [ 'Oblivion Uncut.esp' ]
+    msg:
+      - <<: *patchProvided
+        subs: [ 'Knights of the Nine' ]
+        condition: 'active("Knights - Unofficial Patch.esp") and not file("Oblivion Content Restoration Project -- Knights.esp")'
```

The condition should still check for `Knights.esp` for some edge cases (e.g., a user disabling KotN but forgetting to uninstall its unofficial patch).

```suggestion
        condition: 'active("Knights.esp") and active("Knights - Unofficial Patch.esp") and not file("Oblivion Content Restoration Project -- Knights.esp")'
```